src/Sylius/Bundle/ThemeBundle/composer.json

```diff
@@ -22,8 +22,9 @@
     "require": {
         "php": "^5.5.9|^7.0",
 
-        "sylius/resource": "0.17.*@dev",
+        "sylius/resource-bundle": "^0.17",
 
+        "doctrine/doctrine-module": "^1.0",
```

Comment:
Need to make hydrator independent of this depenency or move `doctrine/orm` from `require-dev` to `require`.